apps/api/src/domain/models/vocabulary-list/entities/value-and-display.entity.ts

```diff
@@ -1,6 +1,7 @@
+import { IValueAndDisplay } from '@coscrad/api-interfaces';
 import { ApiProperty } from '@nestjs/swagger';
 
-export class ValueAndDisplay<T> {
+export class ValueAndDisplay<T> implements IValueAndDisplay<T> {
```

Comment:
Note that we eventually may want to call this `ValueAndLabel`. However, this is not a big issue as we are going to want to map the `VocabularyListVariables` to a dynamic form in the view models layer.